src/input/Input.js

```diff
@@ -172,6 +178,11 @@ Input.propTypes = {
   labelStyle: TextPropTypes.style,
   labelProps: PropTypes.object,
   theme: PropTypes.object,
+  renderErrorMessage: PropTypes.bool,
+};
+
+Input.defaultProps = {
```

Comment:
@flyingcircle thanks for pointing out actually defaultProps was already defined and my defined default props was overriding with that

Will update the PR

Reply:
@flyingcircle I have updated the PR now it will work